vtex/billing-app
# Check LM resources access on install/uninstall

#### What is the purpose of this pull request?
Use actual user headers on access to Apps install/uninstall resources

#### What problem is this solving?
Any user could install apps, because the billing app's token was being used on its Apps' client ➿ 

#### How should this be manually tested?
1. Clone/pull this repo and link it in your workspace
2. Test installing app with user with weak IO access (without the Install Apps resource)
3. A 403 error should be thrown

#### Types of changes
